### PR TITLE
[akamai.siem] don't fail on headers without values

### LINKED ISSUE #9484: [akamai.siem] Failure parsing headers without a value
This error happens when processor raw request headers that doesn't have a value.

> field [_tmp.request.headers] does not contain value_split[: ]

An example value that causes the error is:

`Host: example.com\r\njsessionid:\r\nAccept-Encoding: gzip\r\n`

This happens very frequently. In one environment I observed 510k of this error.message during 24h (and 12.3M w/o errors).

### PR DESCRIPTION
**Pull request by andrewkroh**
## Proposed commit message

Make the kv processor not fail when it encounters an http header without a value (e.g. `jsessionid:\r\n`). In the value_split, this uses a regex to make the space after the colon optional.

Closes #9484

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

File: packages/akamai/data_stream/siem/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -87,24 +87,26 @@ processors:
...
       ignore_missing: true
   - kv:
-      field: _tmp.response.headers
+      if: ctx.json?.httpMessage?.responseHeaders != ""
+      tag: kv_httpMessage_responseHeaders
+      field: json.httpMessage.responseHeaders
       target_field: akamai.siem.response.headers
       field_split: '\r\n'
-      value_split: ': '
+      value_split: ':\s?'
```
Comment (efd6) [inline packages/akamai/data_stream/siem/elasticsearch/ingest_pipeline/default.yml:97]:
```suggestion
      value_split: ':\s*'
```
(same below)

File: packages/akamai/data_stream/siem/elasticsearch/ingest_pipeline/default.yml [hunk 2]
```diff
@@ -87,24 +87,26 @@ processors:
...
       ignore_missing: true
   - kv:
-      field: _tmp.request.headers
+      if: ctx.json?.httpMessage?.requestHeaders != ""
+      tag: kv_httpMessage_requestHeaders
+      field: json.httpMessage.requestHeaders
       target_field: akamai.siem.request.headers
       field_split: '\r\n'
-      value_split: ': '
+      value_split: ':\s?'
```
Comment (andrewkroh) [inline packages/akamai/data_stream/siem/elasticsearch/ingest_pipeline/default.yml:109]:
```suggestion
      value_split: ':\s*'
```

File: packages/akamai/data_stream/siem/elasticsearch/ingest_pipeline/default.yml (hunk 1 above)
Comment (andrewkroh) [reply packages/akamai/data_stream/siem/elasticsearch/ingest_pipeline/default.yml]: Added in 192d39d and 8f0ff22

**Review by efd6 — APPROVED** [review]

File: packages/akamai/data_stream/siem/elasticsearch/ingest_pipeline/default.yml (hunk 1 above)
Comment (efd6) [reply packages/akamai/data_stream/siem/elasticsearch/ingest_pipeline/default.yml]: FWIW It's hard to find a definitive BNF description of this in an RFC, but Apache write their definition like this IBM include no space, and the Go code for parsing MIME headers tolerates `[\s\t]*`.

**Pull request merged**